backend/graminsta/post/services.py

```diff
@@ -0,0 +1,37 @@
+"""
+Service functions for post module
+"""
+
+import datetime
+from django.contrib.auth import get_user_model
+from .models import Post
+
+
+def create_post(publisher_id, description, img):
```

Comment:
I think to create a `Post`, you need `publisher_id`, `description`, `mention_user`, `image`

Reply:
Added